dts/arm/nuvoton/npcx.dtsi

```diff
@@ -93,6 +93,12 @@
 			label = "SCFG";
 		};
 
+		mdc: mdc@4000c000 {
+			compatible = "nuvoton,npcx-mdc";
+			reg = <0x4000c000 0x4>;
```

Comment:
Does the range cover the registers which you want to use?

Reply:
I see, I misread the datasheet, the size here should be 16.